src/workflows/airqo_etl_utils/utils/common_utils.py

```
@@ -59,12 +58,15 @@ def get_country_boundaries(country: str) -> Dict[str, Any]:
 
         Raises:
             IndexError: If the API response does not contain valid bounding box data.
-            requests.RequestException: If the API request fails.
+            requests.exceptions.RequestException: If the HTTP request fails.
         """
-        response = requests.get(
-            f"https://nominatim.openstreetmap.org/search?q={country}&format=json"
+        from airqo_etl_utils.http_client import HttpClient
+
+        data = HttpClient().get_json(
+            f"https://nominatim.openstreetmap.org/search",
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)